pystruct/learners/n_slack_ssvm.py

```diff
@@ -307,10 +311,14 @@ def fit(self, X, Y, constraints=None, warm_start=None, initialize=True):
                     X_b = X[batch]
                     Y_b = Y[batch]
                     indices_b = indices[batch]
-                    candidate_constraints = Parallel(
-                        n_jobs=self.n_jobs, verbose=verbose)(
+                    if any([self.n_jobs == 1, self.pool == None]):
```

Comment:
yes. It is not that big but if you can think of a nice refactoring that would be welcome ;)